oauth2_http/java/com/google/auth/oauth2/GoogleAuthUtils.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2015, Google Inc. All rights reserved.
```

Comment:
nit: 2023

Reply:
please double check if contents are the same in more recent examples. Should be the same

Reply:
Using [statement](https://github.com/googleapis/google-auth-library-java/blob/6c2be858c4024cb0f14ab927af5ff388825cb3b3/oauth2_http/java/com/google/auth/oauth2/GdchCredentials.java#L1-L30) from `GdchCredentials.java` (2022)